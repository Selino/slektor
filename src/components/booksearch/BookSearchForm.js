import React from "react"
import { Button } from "react-bootstrap"
export default function BookSearchForm(props) {
  const handleSubmit = e => {
    e.preventDefault()
    console.log("test")
  }
  return (
    <form
      onSubmit={handleSubmit}
      className='search-form'
      style={{ marginBottom: "1rem" }}
    >
      <input
        type='text'
        placeholder='Enter title'
        name='Search'
        style={{ width: "60%" }}
      />
      <Button variant='primary' type='Submit' size='sm'>
        Search
      </Button>
    </form>
  )
}
