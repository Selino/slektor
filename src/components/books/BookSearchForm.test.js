import React from "react"
import { shallow } from "enzyme"
import BookSearchForm from "./BookSearchForm"

test("should render the book search form", () => {
  const wrapper = shallow(<BookSearchForm />)
  expect(wrapper).toMatchSnapshot()
})
