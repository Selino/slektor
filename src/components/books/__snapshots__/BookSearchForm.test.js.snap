// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the book search form 1`] = `
<form
  className="search-form"
  onSubmit={[Function]}
  style={
    Object {
      "marginBottom": "1rem",
    }
  }
>
  <input
    name="Search"
    placeholder="Enter title"
    style={
      Object {
        "width": "60%",
      }
    }
    type="text"
  />
  <Button
    active={false}
    disabled={false}
    size="sm"
    type="Submit"
    variant="primary"
  >
    Search
  </Button>
</form>
`;
